asserts/database.go

```diff
@@ -108,3 +120,44 @@ func (db *Database) ImportKey(authorityID string, privKey *packet.PrivateKey) (f
 	}
 	return fingerp, nil
 }
+
+// use a generalized matching style along what PGP does where keys can be
+// retrieved by giving suffixes of their fingerprint
```

Comment:
Should we wait until we have a better understanding for introducing it, as it weakens the matching logic and thus the involved security in a way we can't yet foresee, since it depends on how and where this will be used.